backend/requirements.txt

```diff
@@ -1,4 +1,6 @@
+cffi>=1.4.0
 Cython==0.29.3
+enum34
```

Comment:
versions for these packages?

Reply:
I need to check them on the working test deployment and freeze them.

Reply:
done. Resolving.